guides/common/modules/proc_restrict-custom-repository-RHEL9.adoc

```diff
@@ -0,0 +1,9 @@
+[id="Restricting_Custom_Repository_to_RHEL9{context}"]
+= Restricting Custom Repository Sets to RHEL 9 in {Project}
+
+Within a Product where a user is preparing to add repositories to that product, the user can restrict the repositories by operating system. Under Sync Settings, use the Restrict to OS version list to include RHEL operating systems for their corresponding RHEL repositories.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Products* and click on the desired product under *Name*.
```

Comment:
We never ever use desire. 
```suggestion
. In the {ProjectWebUI}, navigate to *Content* > *Products* and click the product that you want to restrict.
```
There's no desire in tech docs. 
IBM Style has the following entry for desire: 

desire v
Do not use. Use “want.”

pp 321

We also never use "under" "over" etc. IBM Style describes exactly how to approach these scenarios.